include/parallel/parallel_implementation.h

```diff
@@ -3191,6 +3191,44 @@ void Communicator::scatter(const std::vector<T> & data,
 
 
 template <typename T>
+void Communicator::scatter(const std::vector<std::vector<T> > & data,
+                           std::vector<T> & recv,
+                           const unsigned int root_id,
+                           const bool identical_buffer_sizes) const
+{
+  if (this->size() == 1)
+    {
+      libmesh_assert (!this->rank());
+      libmesh_assert (!root_id);
+      return;
```

Comment:
Yes...  I missed that on the other implementations too. Too much copy and paste.